[2022-06-27 13:32:31,275] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_B scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2022-06-27 13:32:31,315] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_B scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2022-06-27 13:32:31,317] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:32:31,318] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:32:31,319] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:32:31,428] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_B> on 2022-04-29 00:00:00+00:00
[2022-06-27 13:32:31,441] {standard_task_runner.py:52} INFO - Started process 57932 to run task
[2022-06-27 13:32:31,458] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_B', 'scheduled__2022-04-29T00:00:00+00:00', '--job-id', '584', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpuh8nnjsu', '--error-file', '/tmp/tmpbwopsd5b']
[2022-06-27 13:32:31,459] {standard_task_runner.py:80} INFO - Job 584: Subtask model_B
[2022-06-27 13:32:31,671] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_B scheduled__2022-04-29T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:32:32,128] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_B
AIRFLOW_CTX_EXECUTION_DATE=2022-04-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-29T00:00:00+00:00
[2022-06-27 13:32:32,131] {python.py:173} INFO - Done. Returned value was: 2
[2022-06-27 13:32:32,228] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_B, execution_date=20220429T000000, start_date=20220627T133231, end_date=20220627T133232
[2022-06-27 13:32:32,397] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:32:32,636] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
